RELEASING.md

```diff
@@ -24,3 +24,11 @@ make release
 Once a release is prepared 2 Pull Request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
 
 > **_NOTE:_** Please review the changes in case there are errors if the PR were modified after the automation ran.
+
+#### Versioning standard
+
+Releases should follow the following convention for versioning numbers. It's responsibility of the committer to update the version number within the same pull request where the change was introduced when the changes affect `minor` or `major` as follows:
```

Comment:
I think you should at least point this to https://semver.org/ and optionally copy-paste their version. So I would start with "This project follows [semver](https://semver.org/).".